Cartfile.resolved

```diff
@@ -1,3 +1,3 @@
-binary "https://cirrusmd-ios-sdk-release.s3.amazonaws.com/CirrusMDSDK.json" "1.4.3"
+binary "https://cirrusmd-ios-sdk-release.s3.amazonaws.com/CirrusMDSDK.json" "1.5.0"
```

Comment:
What do you think about gitignoring this file? That way, we don't have to update the example app for every release. Ideally, there's an automated test that runs to at least build the example app with the latest SDK for both pods and carthage. But, that's not setup yet.